bin/dlang/install

```diff
@@ -9,22 +9,41 @@ set -eu
 
 beaver=$(dirname $0)/../beaver
 
+remove_old_dmd=0
+
 # Package name deduced based on supplied DMD version
 case "$DMD" in
     dmd*   ) PKG= ;;
     1.*    ) PKG="dmd1=$DMD-$DIST" ;;
     2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;;
-    2.*    ) PKG="dmd-bin=$DMD libphobos2-dev=$DMD" ;;
+    2.*    ) if [ $(echo $DMD | cut -d. -f2) -ge 077 ]
```

Comment:
Now seeing it, I don't know how `test` deals with leading zeroes, have to tried this?

Reply:
Yep, tested with few versions - seems to ignore zeroes but I kept one "just in case".